validity_rollups_on_bitcoin.md

```diff
@@ -647,6 +746,20 @@ Proofs of the validity of the blockchain solve the IBD issue for blocks of arbit
 
 This "proof-sync" technique may solve the IBD problem of large blocks, but it does not solve the mining centralization problems. Work still needs to be done to decrease the bandwidth costs and propogation times of increased Layer 1 transaction volume, for example by implementing or further optimizing protocols such as Compact Block Relay and Erlay that reduce bandwidth requirements even as transaction volume increases.[132, 133] And there's no getting around the requirement for storage of blockchain data, which increases as the size of the blockchain grows. Still, having one less problem to solve (the IBD problem) is a nice improvement. The best part is that no consensus rule changes are required to implement and start using proof-sync clients. As of the time of writing, Lukas George and Robin Linus have begun work on a proof-sync bitcoin client implementation written in Cairo called "Zerosync".[134] 
```

Comment:
This would be really cool to expand on if you think it fits.  I'm assuming they build some kind of state commitment?  Is this generic enough that we could eventually soft fork in the state roots?  Although since it's not actually a consensus change I suppose you could gossip the state roots and do it optimistically/incrementally like utreexo sync.

Reply:
I wish I could! But I don't know exactly how they work, and I think it's too early and too tangential for this report to spend any more time on the topic than I have already. Given that there are two "STARK-verified bitcoin" projects that I'm aware of (Zerosync and Khepri) it's [only a matter of time until some long-form documentation comes out](https://twitter.com/dimahledba/status/1565382982382272513) about them. I appreciate the encouragement but won't be expanding on this concept any further in this report :)